test/e2e/ga.test.js

```diff
@@ -0,0 +1,97 @@
+// Modules, constants, and variables
+// npm run test:e2e ga.test.js
+// -----------------------------------------------------------------------------
+const docsifyInit = require('../helpers/docsify-init');
+const { test, expect } = require('./fixtures/docsify-init-fixture');
+
+const gaTagList = [
+  'AW-YYYYYY', // Google Ads
+  'DC-ZZZZZZ', // Floodlight
+  'G-XXXXXX', // Google Analytics 4 (GA4)
+  'UA-XXXXXX', // Google Universal Analytics (GA3)
+];
+
+// Suite
+// -----------------------------------------------------------------------------
+test.describe('GA Plugin Tests', () => {
+  // page request listened, print collect url
+  function pageRequestListened(page) {
+    // page.on('request', request => {
+    //   if (request.url().indexOf('www.google-analytics.com') !== -1) {
+    //     console.log(request.url());
+    //   }
+    // });
+
+    page.on('response', response => {
+      const request = response.request();
+      // googleads.g.doubleclick.net
+      // www.google-analytics.com
+      // www.googletagmanager.com
+      const reg =
+        /googleads\.g\.doubleclick\.net|www\.google-analytics\.com|www\.googletagmanager\.com/g;
+      if (request.url().match(reg)) {
+        console.log(request.url(), response.status());
```

Comment:
Let's remove or comment out the `console.log` tests so we don't see them in every test run.